### PR TITLE
Add remove book

### PR DESCRIPTION
**Pull request by TimmyChan99**
In this tasks I :

- Configure the Redux Store 
-  Write actions: `ADD_BOOK` and `REMOVE_BOOK`
-  Create a reducer for adding and removing books
- Get the book info from the form and tor it in `booksReducer`
- Add functionality to submit button in the form and to remove button
- Display the books list

### CODE REVIEW

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Fatima, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
- [ ] the readme file in your repo doesn't comply with the general Microverse requirements, please use [the template](https://github.com/microverseinc/readme-template) for the readme file provided by Microverse.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.js [hunk 1]
```diff
@@ -1,17 +1,53 @@
...
+        <input name="title" type="text" value={title} onChange={handleChange} />
+        <select name="category" id="categories" onChange={handleChange}>
+          <option value="none">Category</option>
+          <option value="Action">Action</option>
+          <option value="Science Fiction">Science Fiction</option>
+          <option value="Economy">Economy</option>
+        </select>
+        <button type="submit">ADD BOOK</button>
+      </form>
+    </>
```
Comment (er-contreras) [inline src/components/Form.js]: Sorry that the last CR didn't address this but is necessary to complete your project.

- [ ] In your React component responsible for adding new books set the data inputs in the local React state (set title and author).
* It's a requirement to also add an input to add an author and render the name of the author. 😉

**Review by er-contreras — CHANGES_REQUESTED** [review]
Hi @TimmyChan99, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

almost there

## Required Changes ♻️

- [ ] Your MIT link in the readme file is broken please fix it.
- [ ] In the readme don't forget to mention that people should install 'npm install' packages to work with your project. 
 
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Form.js (hunk 1 above)
Comment (rloterh) [reply src/components/Form.js]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Fatima, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:. Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have no more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**